lib/spack/spack/build_systems/cmake.py

```diff
@@ -49,6 +50,8 @@ class CMakePackage(PackageBase):
     # build-system class we are using
     build_system_class = 'CMakePackage'
 
+    spack.directives.depends_on('cmake', type='build')
```

Comment:
I think the narrow import is good but can you do `from spack.directives import depends_on` so this is readable?